ui-integration-test-lib/build.gradle.kts

```diff
@@ -0,0 +1,70 @@
+plugins {
+  id("com.android.library")
+  kotlin("android")
+  id("zcash.android-build-conventions")
+  id("wtf.emulator.gradle")
+  id("zcash.emulator-wtf-conventions")
+}
+
+// Force orchestrator to be used for this module, because we need cleared state to generate screenshots
+val isOrchestratorEnabled = true
+
+android {
+  defaultConfig {
+    if (isOrchestratorEnabled) {
+      testInstrumentationRunnerArguments["clearPackageData"] = "true"
+    }
+
+    testInstrumentationRunner = "co.electriccoin.zcash.test.ZcashUiTestRunner"
+  }
+
+  if (isOrchestratorEnabled) {
+    testOptions {
+      execution = "ANDROIDX_TEST_ORCHESTRATOR"
+    }
+  }
+
+  buildFeatures {
+    compose = true
+  }
+
+  composeOptions {
+    kotlinCompilerExtensionVersion = libs.androidx.compose.compiler.get().versionConstraint.displayName
+  }
+}
+
+dependencies {
+  androidTestImplementation(projects.uiLib)
+  androidTestImplementation(projects.uiDesignLib)
+  androidTestImplementation(projects.testLib)
+
+  androidTestImplementation(libs.bundles.androidx.test)
+  androidTestImplementation(libs.bundles.androidx.compose.core)
+
+  androidTestImplementation(libs.androidx.compose.test.junit)
+  androidTestImplementation(libs.androidx.navigation.compose)
+  androidTestImplementation(libs.androidx.uiAutomator)
+
+  if (isOrchestratorEnabled) {
+    androidTestUtil(libs.androidx.test.orchestrator) {
+      artifact {
+        type = "apk"
+      }
+    }
+  }
+}
+
+// This block exists to override the minimum SDK version for our integration tests from the property
+// ANDROID_LIB_MIN_SDK_VERSION to ANDROID_APP_MIN_SDK_VERSION, as by our integration tests we aim to
+// test the UI of the app.
+emulatorwtf {
```

Comment:
I used this block to let our integration tests run on our App minimum SDK version. Also, these tests didn't work well on API 23. What do you think about it?

Reply:
We’re only using 23 because wtf doesn’t have 24 set up yet but they’re working on it this week.

Do these tests work better on 24?

My hope was to try to keep our modules on ~24 so we have flexibility but have the app somewhere in the $current-5 range so that we’re only supporting more modern devices that have secure elements and get security patches.

Reply:
Ah, understand. It's good to have the 24 available for us soon, hopefully. On the 24, our tests went well. I filed an issue on updating the minimal version for us, also added to the codebase. https://github.com/zcash/secant-android-wallet/issues/448

Reply:
We should be able to switch this to 24 in the `build-conventions` now.  totally OK if you want to do this as part of this PR or defer it.